packages/template-retail-react-app/app/pages/login/index.jsx

```diff
@@ -45,7 +45,7 @@ const Login = () => {
     useEffect(() => {
         if (customer.authType != null && customer.isRegistered) {
             if (location?.state?.directedFrom) {
-                navigate(location.state.directedFrom)
+                navigate(removeSiteLocaleFromPath(location.state.directedFrom))
```

Comment:
@sandragolden Have you thought about adding this logic directly into the `navigate` function. I feel like that would be a little more future proof encase there are other occurrences like this or ones partners might run into.

Reply:
great idea, thanks @bendvc! i made the update in `use-navigation` instead, let me know if this is what you had in mind